test/routing/FormFooter.test.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { Route } from 'react-router-dom'
+
+import FormRouter from '../../src/routing/Router';
+import FormFooter from '../../src/routing/FormFooter';
+import { Page, TextField } from '../../src';
+
+
+describe('FormTitle', () => {
```

Comment:
Can you change the describe message to be 'FormFooter' so that it doesn't get mixed up with the FormTitle tests?